site/static/js/plugins.js

```diff
@@ -134,6 +134,20 @@ function processAsciinemaTags(content) {
   );
 }
 
+// Workaround for https://github.com/docsifyjs/docsify/pull/1468
+function defaultLinkTargets() {
```

Comment:
I'm not sure I understand why this is needed. Can you help me understand?

Reply:
External links (such as those in kpt.dev/contact/) open in a new frame if you first navigate directly to the referrer, then click them, as expected. However, there's currently a bug where going to another page first (say, kpt.dev/faq/) then the referrer will drop the target, changing it to _self as described in the commented PR. Without explicitly marking those links as safe for cross-origin, links targeting the same frame as a Docsify page won't open in the routing mode we use. This function resets the target of each external page link that isn't marked as cross-origin safe to whatever Docsify thinks it should've been normally (blank in our case)